File: 00_ETC/5789.cpp
#include <iostream>
using namespace std;

int main() {
    cin.tie(0)->sync_with_stdio(0);

    int n;
    cin >> n;
    for (int i = 0; i < n; ++i) {
        string s;
        cin >> s;

        int middle = s.length() / 2;
        string sub = s.substr(middle - 1, 2);
        char first = sub.front();
        char last = sub.back();
        if (first == last) {
            cout << "Do-it" << endl;
        } else {
            cout << "Do-it-Not" << endl;
        }
    }
    return 0;
}
//
// Created by 최승우 on 25. 9. 24.
//

File: 00_ETC_PY/5789.py
n = int(input())
for i in range(n):
    s = input()

    middle = len(s) // 2 - 1
    sub = s[middle:middle + 2]
    first = sub[0]
    last = sub[-1]
    if first == last:
        print("Do-it")
    else:
        print("Do-it-Not")